Drivers/BynderFieldDisplayDriver.cs

```diff
@@ -32,7 +32,7 @@ public class BynderFieldDisplayDriver : ContentFieldDisplayDriver<BynderField>
     public override IDisplayResult Display(BynderField field, BuildFieldDisplayContext fieldDisplayContext) =>
         Initialize<BynderFieldDisplayViewModel>(
             GetDisplayShapeType(fieldDisplayContext),
-            viewModel => viewModel.Resources.AddRange(field.Resources))
+                viewModel => viewModel.Resources.ToList().AddRange(field.Resources))

```

Comment:
```suggestion
            viewModel => viewModel.Resources.ToList().AddRange(field.Resources))
```

Formatting problems. Also inside the `Edit()` and inside `UpdateAsync()` method in this file and also in other files. Please look at the analyzer warnings: 

![image](https://user-images.githubusercontent.com/80963259/159969185-3ad5b920-22fa-4896-a0d2-e05522bb90dc.png)

Also, see: [General code conventions and development practices.](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/28213305/General+code+conventions+and+development+practices#:~:text=be%20sure%20to%20format%20the%20code%20according%20to%20the%20conventions%20mentioned.%20in%20visual%20studio%20most%20of%20the%20formatting%20can%20be%20automatically%20applied%20with%20ctrl%2Bk%2Bd%20but%20also%20do%20automatically%20on%20save%3A)